src/main/java/org/jabref/gui/actions/MassSetFieldAction.java

```diff
@@ -281,6 +301,31 @@ private static UndoableEdit massSetField(Collection<BibEntry> entries, String fi
     }
 
     /**
+     * Append a given value to a given field for all entries in a Collection. This method DOES NOT update any UndoManager,
+     * but returns a relevant CompoundEdit that should be registered by the caller.
+     *
+     * @param entries         The entries to process the operation for.
+     * @param field           The name of the field to append to.
+     * @param text            The value to set. A null in this case will simply preserve the current field state.
+     * @return A CompoundEdit for the entire operation.
+     */
+    private static UndoableEdit massAppendField(Collection<BibEntry> entries, String field, String text) {
+
+        if (text == null) {
+            text = "";
+        }
+
+        NamedCompound ce = new NamedCompound(Localization.lang("Append field"));
```

Comment:
Since this is new code, please do proper naming (and don't get inspired by all the bad naming in JabRef). So please call this `compoundEdit` (or something similar) instead of `ce`.